front/app/containers/Admin/sideBar/UserMenu.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+// components
+import { Box, Icon, Text } from '@citizenlab/cl2-component-library';
+import { Popup } from 'semantic-ui-react';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// style
+import { colors } from 'utils/styleUtils';
+import { ItemMenu, StyledBox } from './styles';
+
+// hooks
+import useAuthUser from 'hooks/useAuthUser';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import Avatar from 'components/Avatar';
+
+// services
+import { signOut } from 'services/auth';
+
+export const UserMenu = () => {
+  const { formatMessage } = useIntl();
+  const authUser = useAuthUser();
+
+  if (isNilOrError(authUser)) {
+    return null;
+  }
+
+  return (
+    <Popup
+      trigger={
+        <StyledBox
+          as="button"
+          width="100%"
+          display="flex"
+          justifyContent="flex-start"
+          mb="25px"
+        >
+          <Box display="flex" alignItems="center">
+            <Avatar userId={authUser.id} size={30} addVerificationBadge />
+            <Box display="flex" flex="1" flexDirection="column" opacity={0.7}>
+              <Text
+                color="white"
+                my="0px"
+                fontSize="base"
+              >{`${authUser.attributes.first_name} ${authUser.attributes.last_name}`}</Text>
+            </Box>
+            <Box mr="20px">
+              <Icon name="chevron-down" fill={colors.white} />
+            </Box>
+          </Box>
+        </StyledBox>
+      }
+      on="click"
+      position="right center"
+      positionFixed
+      offset={[0, -40]}
```

Comment:
Not sure at all, but maybe moving the popup to the right could make it look a bit better? 
![image](https://user-images.githubusercontent.com/91883434/234260462-fc9726fb-ff85-4a15-9187-f726923c5b7a.png)

Reply:
@alexander-cit do you mean just after the sidebar?

Reply:
Yes, closer to the sidebar border. But it's up to you.